applications/GeoMechanicsApplication/custom_constitutive/coulomb_with_tension_cut_off_impl.cpp

```diff
@@ -24,6 +24,18 @@ namespace
 
 using namespace Kratos;
 
+const double CalculatePlasticMultiplier(const Vector& rSigmaTau,
+                                        const Vector& rDerivativeOfFlowFunction,
+                                        double        FrictionAngleInRadians,
+                                        double        Cohesion)
+{
+    const auto sin_phi   = std::sin(FrictionAngleInRadians);
+    const auto numerator = sin_phi * rDerivativeOfFlowFunction[0] + rDerivativeOfFlowFunction[1];
+    const auto lambda =
+        (Cohesion * std::cos(FrictionAngleInRadians) - rSigmaTau[0] * sin_phi - rSigmaTau[1]) / numerator;
+    return lambda;
```

Comment:
Let's combine these two statements:
```suggestion
    return (Cohesion * std::cos(FrictionAngleInRadians) - rSigmaTau[0] * sin_phi - rSigmaTau[1]) / numerator;
```